types/html-webpack-tags-plugin/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for html-webpack-tags-plugin 2.0
+// Project: https://github.com/jharris4/html-webpack-tags-plugin
+// Definitions by: // Definitions by: 贺师俊 <https://github.com/hax>, Vladimir Grenaderov <https://github.com/VladimirGrenaderov>, Max Boguslavskiy <https://github.com/maxbogus>
```

Comment:
You have duplicated the `// Definitions by:` part of the comment. I would also suggest this:

```suggestion
// Definitions by: 贺师俊 <https://github.com/hax>
//                 Vladimir Grenaderov <https://github.com/VladimirGrenaderov>
//                 Max Boguslavskiy <https://github.com/maxbogus>
```